### ISSUE TITLE
Data Input Action

### ISSUE THREAD

**Issue by wsguede** [Jmgr/actiona#56]
Not a defect, this is already working as expected (probably). -- So this would be a feature add.

If I click close or cancel on the input dialog, it would be nice to be able to set that action inside the data input edit action setup.
ie -- Cancel button action --> (could be drop down with some basic choices... Stop script, continue with default, roll your own...)

--- Same goes for clicking the [x] to close the dialog.

**Comment by Jmgr** [comment]
Good point! Thanks for the suggestion, and sorry for the lack of answer!
